### PR TITLE
[codex] Surface area-fill crop movement cues

### PR DESCRIPTION
**Pull request by ebelo**
## Summary

- add report-level crop color movement cues to the style-audit area-fill table
- add a per-camera area-fill focus table that filters style-audit candidates to each cropped camera's zoom
- document the new area-fill crop context and cover it in visual crop report tests

## Why

The current #949 Zermatt manual crops show a strong `lighter + blue higher` movement while path/pedestrian stroke, dash, cap, and join cues do not point to a direct stroke-width fix. This keeps the landcover/tint area-fill candidates visible beside the crop-local movement without reopening the full style audit.

Validation artifact regenerated from existing token-free inputs:

- `debug/mapbox-outdoors-visual-crops/20260521T194828Z/summary.md`

## Validation

- `python3 -m pytest tests/test_mapbox_outdoors_visual_crops.py -q --tb=short`
- `python3 -m pytest tests/ -x -q --tb=short`
- `git diff --check`

Refs #949.

### CODE REVIEW

File: validation/mapbox_outdoors_visual_crops.py [hunk 1]
```diff
@@ -1130,6 +1145,92 @@ def _style_audit_filter_signature_rows(
...
+) -> bool:
+    if camera_zoom is None:
+        return True
+    min_zoom, max_zoom = _style_audit_candidate_zoom_bounds(
+        candidate,
+        style_audit_layer=style_audit_layer,
+    )
+    if min_zoom is not None and camera_zoom < min_zoom:
+        return False
+    return not (max_zoom is not None and camera_zoom >= max_zoom)
```
[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_visual_crops.py]: Fixed in 1570c97. The zoom-band parser now distinguishes explicit `z<=N` inclusive upper bounds from `z<N`/range/maxzoom-exclusive bounds, and `tests/test_mapbox_outdoors_visual_crops.py` covers the exact boundary case.

[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_visual_crops.py]: Fixed in 1570c97 by tracking inclusive upper zoom bands separately from Mapbox-style exclusive maxzoom bands and adding regression coverage for `z<=N` candidates at the boundary camera zoom.

File: validation/mapbox_outdoors_visual_crops.py [hunk 2]
```diff
@@ -1178,15 +1279,170 @@ def _style_audit_area_fill_focus(
...
+                section,
+                summary,
+                camera_zoom=camera_zoom,
+                qgis_converter_warnings_by_layer=qgis_converter_warnings_by_layer,
+                style_audit_layers_by_id=style_audit_layers_by_id,
+                sample_limit=sample_limit,
+            )
+        )
+        is not None
+    ]
```
[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_visual_crops.py:1407]: Fixed in a7d7a96 by capturing the crops list once before computing `crop_count`.

File: validation/mapbox_outdoors_visual_crops.py [hunk 3]
```diff
@@ -1178,15 +1279,170 @@ def _style_audit_area_fill_focus(
...
+    if not area_rows:
+        return None
+    return {
```
[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_visual_crops.py:1410]: Fixed in a7d7a96 by adding a `Crops` column to the area-fill camera-focus Markdown table, so the stored `crop_count` is visible in the report.

File: validation/mapbox_outdoors_visual_crops.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_visual_crops.py:1407]: Fixed in a7d7a96 by capturing `crops = camera.get("crops")` once before computing `crop_count`.

File: validation/mapbox_outdoors_visual_crops.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_visual_crops.py:1410]: Fixed in a7d7a96 by rendering the stored `crop_count` as a `Crops` column in the area-fill camera focus table and adding a markdown assertion for it.
